pages/api/prospect/create.tsx

```diff
@@ -0,0 +1,46 @@
+import prisma from '../../../lib/prisma'
+import validateEmail from 'email-validator'
+ 
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  // get the prospect's email and phone number
+  // and trim any new line characters and spaces
+  const prospectEmail = req.body.mail.trim().replace(' ', '')
+  const prospectPhone = req.body.phone.trim().replace(' ', '')
+
+  // if the prospect's email already exists 
+  // or is invalid, return an error
+  if (await isEmailInUse(prospectEmail)) {
+    return await res.status(400).send({ error: 'Email is already in use!' })
+  } else if (!validateEmail.validate(prospectEmail)) {
+    return await res.status(400).send({ error: 'Invalid email adress!' })
+  }
+
```

Comment:
let's validate phone number as well

Reply:
Can you please give me more details?
How exactly should we validate a phone number?

Reply:
check if there is a library for it and use that